### ISSUE TITLE
🐛 Bug: Accessibility issue on mochajs.org: scrollable region missing keyboard focus

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [mochajs/mocha#5067]
### Bug Report Checklist

- [X] I have read and agree to Mocha's [Code of Conduct](https://github.com/mochajs/mocha/blob/master/.github/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/mochajs/mocha/blob/master/.github/CONTRIBUTING.md)
- [X] I have searched for [related issues](https://github.com/mochajs/mocha/issues?q=is%3Aissue) and [issues with the `faq` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Afaq%20), but none matched my issue.
- [X] I have 'smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, my usage of Mocha, or Mocha itself.
- [X] I want to provide a PR to resolve this

### Expected

The website should not have any accessibility issues - including ones detectable by the [axe browser devtools](https://www.deque.com/axe/browser-extensions).

### Actual

The `const assert = require('assert');` code block under https://mochajs.org/#delayed-root-suite is a [scrollable region without keyboard focus](https://dequeuniversity.com/rules/axe/4.8/scrollable-region-focusable), making it unusable for users who scroll with non-mouse devices - such as keyboards.

### Minimal, Reproducible Example

1. Visit https://mochajs.org
2. Wait for the page to load & fade in
3. Run the [axe browser devtools](https://www.deque.com/axe/browser-extensions)

### Versions

n/a

### Additional Info

https://web.dev/explore/accessible explains more around web accessibility.

I first ran a [Lighthouse accessibility audits](https://developer.chrome.com/docs/lighthouse/accessibility/scoring) but it didn't wait for the page to fade in and reported many false negatives as a result.

**Comment by JoshuaKGoldberg** [comment]
The site's tooling is a little old and I'm not sure that it'll be straightforward to fix this until #5071 bumps our site-related devDependencies to latest. Removing `good first issue` and unassigning myself for now.

If someone is familiar with `markdown-it` and `prism`, and knows that there's an easy fix in the older versions we use, please do go ahead and fix it in the meantime! But otherwise I'll wait to take action till #5071 is in.
